include/RE/A/Actor.cpp

```diff
@@ -280,6 +299,71 @@ namespace RE
 		return base ? base->race : nullptr;
 	}
 
+	TESObjectARMO* Actor::GetSkin(BGSBipedObjectForm::BipedObjectSlot a_slot)
+	{
+		if (a_slot == BGSBipedObjectForm::BipedObjectSlot::kNone) {
+			return nullptr;
+		}
```

Comment:
This isn't papyrus, if your function has preconditions, then `assert` on them. Don't turn this into runtime overhead.